webapp/portlet/src/main/webapp/skin/less/portlet/VerticalMenu/Style.less

```diff
@@ -0,0 +1,34 @@
+/**
+
+  This file is part of the Meeds project (https://meeds.io/).
+
+  Copyright (C) 2020 - 2023 Meeds Association contact@meeds.io
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+*/
+
+@import "../../variables.less";
+@import "../../mixins.less";
+
+.VerticalMenu {
+  overflow-x: hidden;
+  overflow-y: auto;
+  min-height: 100vh;
+  max-height: 100vh;
+  max-width:350px !important;
+  min-width:350px !important;
+  left: 0 !important;
+  top: 0 !important;
+  position: fixed !important;
+}
```

Comment:
These classes are too specific for certain elements and do not exist in helpers or Vuetify classes. 
it is similar to hamburger menu menu classes with some changes

Reply:
ok, you can manage to use CSS helpers with this. in fact, there will be a problem by example when using viewport in mobile... but I will not block the merge for now and I will attempt to clean this up after.
( BTW, the HamburgerMenu is old and shouldn't be used as a reference, as you may know the Best Practices had evolved **since 6.0**)